flare/output.py

```diff
@@ -134,6 +136,55 @@ def write_header(
         f = logging.getLogger(self.basename + "log")
         f.info(f"{datetime.datetime.now()}")
 
+        # Write package version numbers
```

Comment:
Good idea. Can you add this to the PR description?

Reply:
added